docs/astro/sql/operators/raw_sql.rst

```diff
@@ -0,0 +1,27 @@
+====================
+run_raw_sql operator
+====================
+
+It allows users to declare any SQL statement using the Astro SDK templating available in transform. By default this operator returns ``None`` but it also empowers users to define the task output (e.g. a list with the row results, etc) by using the handler argument.
```

Comment:
Also, I think it is worth explaining the templating syntax and context available for templates.

Reply:
Can you please elaborate a little bit? 
Do we template any param for ``RawSQLOperator``?? 
Do we explain param in example Doc I think this gets generated for API doc. WDYT?

Reply:
Added the template section in concepts.rst

Reply:
add reference to that here. thanks @sunank200